jsconfig.json

```diff
@@ -19,7 +19,7 @@
 		"**/dist/*",
 		"cached-results.json",
 		"stats.json",
```

Comment:
This needs to change to `client/stats.json` as well.

Reply:
Good call 👍 Fixed in https://github.com/Automattic/wp-calypso/pull/39317/commits/f7e6a21b23c59adb8c60eccb6b90159991111486